### PR TITLE
Get Users object

### PR DESCRIPTION
**Pull request by petericebear**
This is for getting the forge_token for generating the provisions request later

### CODE REVIEW

**Comment by mpociot** [comment]
It would be better if we handle this like the other API calls and return a `User` object instead of a plain array.

**Comment by petericebear** [comment]
Ah yes. Will change it and create a new pull-request..

**Comment by mpociot** [comment]
You can just update this one if you want 👍

**Comment by petericebear** [comment]
It returns the full User object

File: src/Mpociot/Blacksmith/Models/User.php [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Mpociot\Blacksmith\Models;
+
+class User extends ForgeModel
+{
+    /**
+     * @return array
+     */
+    public function toArray()
```
Comment (djekl) [inline src/Mpociot/Blacksmith/Models/User.php]: Why does this function exist in the `User` model. Its already in the extended `ForgeModel` and does the exact same. Its OK to have an empty model. It looks to me like you have copied the `Server` model, which also needs it removing from.

**Review by djekl — CHANGES_REQUESTED** [review]

File: src/Mpociot/Blacksmith/Models/User.php (hunk 1 above)
Comment (petericebear) [reply src/Mpociot/Blacksmith/Models/User.php]: Good point, I removed it.. Yes I copied it from another file.
Currently working on (custom) server creation function and needed the User object first

**Comment by petericebear** [comment]
@mpociot I added a new function of issue #1 .. Server creation, and updated readme with an example for creation a custom load balancer.

**Comment by mpociot** [comment]
Thank you!

**Pull request merged**